app/views/layouts/home.html.erb

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>RecipeApp</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <%= csrf_meta_tags %>
+    <%= csp_meta_tag %>
+    <%= stylesheet_link_tag "application", "data-turbo-track": "reload" %>
+    <%= javascript_importmap_tags %>
+  </head>
+
+  <body>
+    <!-- Navigation Bar or links to other pages -->
+    <nav>
+      <ul>
+        <li><%= link_to "Home", root_path %></li>
+        <li><%= link_to "Food List", foods_path %></li>
+        <li><%= link_to "Login", new_user_session_path %></li>
+        <li><%= link_to "Register", new_user_registration_path %></li>
+      </ul>
+    </nav>
+
+    <!-- Home Page Content -->
+    <div class="home-content">
+      <%= yield %>
+    </div>
+  </body>
```

Comment:
- [ ] Great progress so far! 👏 However, I'd like to suggest an enhancement. It might be beneficial to designate your home page as one of the main routes, such as `/foods`. This way, when users log in, they won't encounter a plain page like they currently do. Here's the current view after logging in:


<img width="1115" alt="Screenshot 2024-02-07 at 23 36 03" src="https://github.com/over-geek/RecipeApp/assets/107192076/7931b357-bc0c-4730-99cf-2882866a0b42">

Happy coding! ✅😊👨‍💻